src/pages/dar_application/ResearcherInfo.jsx

```diff
@@ -0,0 +1,394 @@
+              ariaLevel={ariaLevel + 2}
+              validation={validation.checkCollaborator}
+              onValidationChange={onValidationChange}
+              onChange={({ key, value }) => formFieldChange({ key, value })}
+              defaultValue={formData.checkCollaborator}
+            />
+          </div>
+        </div>
+
+        <div className='dar-application-row'>
+          <FormField
+            id='piName'
+            disabled={readOnlyMode}
+            description='I certify that the principal investigator listed below is aware of this study'
+            placeholder='Firstname Lastname'
+            title='1.3 Principal Investigator'
+            validators={[FormValidators.REQUIRED]}
+            ariaLevel={ariaLevel + 1}
+            validation={validation.piName}
+            onValidationChange={onValidationChange}
+            onChange={({ key, value }) => formFieldChange({ key, value })}
+            defaultValue={formData.piName}
+          />
+        </div>
+
+        <div className='dar-application-row' data-cy='internal-lab-staff'>
+          <h3>1.4 Internal Lab Staff</h3>
+          {includeInstructions && (
+            <div>
+             Please add internal Lab Staff here. Internal Lab Staff are defined as users of data from
+            this data access request, including any that are downloaded or utilized in the cloud.
+            please do not list External Collaborators or Internal Collaborators at a PI or equivalent
+            level here. If your DAR is approved, you will be responsible for the appropriate use of the
+            data by each individual listed in this section.
+            </div>
+          )}
+          <CollaboratorList
+            formFieldChange={formFieldChange}
+            collaborators={formData.labCollaborators}
+            collaboratorKey='labCollaborators'
+            collaboratorLabel='Internal Lab Member'
+            setCompleted={setLabCollaboratorsCompleted}
+            validation={validation.labCollaborators}
+            onValidationChange={onValidationChange}
+            showApproval={true}
+            disabled={!isEmpty(darCode) || readOnlyMode}
+          />
+        </div>
+
+        <div className='dar-application-row' data-cy='internal-collaborators'>
+          <h3>1.5 Internal Collaborators</h3>
+          {includeInstructions && (
+            <div>
+              {'Please list Internal Collaborators here. Internal Collaborators are defined as individuals who are not under the direct supervision of the PI (e.g., not a member of the PI\'s laboratory) who assists with the PI\'s research project involving controlled-access data subject to the NIH GDS Policy. Internal collaborators are employees of the Requesting PI\'s institution and work at the same location/campus as the PI. Internal Collaborators must be at the PI or equivalent level and are required to have a Library Card and submit their own data access request.'}
```

Comment:
This is correct as a direct translation from the `js` version, but I think it can be better expressed as simple text with html entities instead of the escaped quotes in the various `PI\'s` fragments:
```suggestion
              Please list Internal Collaborators here. Internal Collaborators are defined as individuals who are not
              under the direct supervision of the PI (e.g., not a member of the PI&apos;s laboratory) who assists with
              the PI&apos;s research project involving controlled-access data subject to the NIH GDS Policy. Internal
              collaborators are employees of the Requesting PI&apos;s institution and work at the same location/campus
              as the PI. Internal Collaborators must be at the PI or equivalent level and are required to have a Library
              Card and submit their own data access request.
```